src/bin/cargo/commands/vendor.rs

```diff
@@ -7,59 +7,57 @@ pub fn cli() -> App {
         .about("Vendor all dependencies for a project locally")
         .arg_quiet()
         .arg_manifest_path()
-        .arg(Arg::with_name("path").help("Where to vendor crates (`vendor` by default)"))
         .arg(
-            Arg::with_name("no-delete")
+            Arg::new("path")
+                .allow_invalid_utf8(true)
+                .help("Where to vendor crates (`vendor` by default)"),
+        )
+        .arg(
+            Arg::new("no-delete")
                 .long("no-delete")
                 .help("Don't delete older crates in the vendor directory"),
         )
         .arg(
-            Arg::with_name("tomls")
-                .short("s")
+            Arg::new("tomls")
+                .short('s')
                 .long("sync")
                 .help("Additional `Cargo.toml` to sync and vendor")
                 .value_name("TOML")
-                .multiple(true),
+                .allow_invalid_utf8(true)
+                .multiple_occurrences(true)
+                .multiple_values(true),
         )
```

Comment:
> cargo vendor --sync did not use multi_opt and so it has both
multiple occurrences and multiple values. If we want to deprecate
this, we'll need unstable-grouped to be stablized (or pin our clap
version) and ensure each group has only 1 value.